physionet-django/user/forms.py

```diff
@@ -298,6 +299,13 @@ class RegistrationForm(forms.ModelForm):
                     widget=forms.TextInput(attrs={'class': 'form-control'}),
                     validators=[validate_name])
 
+    privacy_policy = forms.BooleanField(required=True,
+                                        label=format_html(
+                                            'I consent to the collection and use of my personal '
+                                            'information consistent with the '
+                                            f'<a href="{settings.PRIVACY_POLICY_URL}">Privacy Policy</a>. '
+                                            'Without your consent, we can not create an account')
+                                        )
```

Comment:
I imagine certain deployments would want to have a completely ' different wording for the Privacy Policy. We should probably not lock them in.

I'd say the entire label should be somehow configurable. We might also ask Tom about his thoughts on this, since he greenlighted this becoming a generic feature.